research/ingest-experiments/cloud-artifacts/codegen-task_e_69e4477bddd8832eb52be7fadb47be38/extracted/research/ingest-experiments/03-codegen-results.md

```
@@ -0,0 +1,509 @@
+import { parseOtlpToSamples } from '../../packages/o11ytsdb/dist/ingest.js';
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)